octobot_trading/exchanges/exchange_util.py

```diff
@@ -0,0 +1,40 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+
+from octobot_trading.exchanges.margin.margin_exchange import MarginExchange
+from octobot_trading.exchanges.rest_exchange import RestExchange
+
+
+def get_margin_exchange_class(exchange_type):
+    return _search_exchange_class_from_exchange_type(exchange_type, MarginExchange)
+
+
+def get_rest_exchange_class(exchange_type):
+    exchange_class_candidate: RestExchange = _search_exchange_class_from_exchange_type(exchange_type, RestExchange)
```

Comment:
Wont this return ```MarginExchange``` since ```class MarginExchange(RestExchange)```  ?

Reply:
You mean this function should explicit its return type with `MarginExchange` ?

Reply:
I mean ```get_rest_exchange_class()``` will be returning a margin exchange instead of a REST exchange while still being called ```get_rest_exchange_class()```. To me it's a bit confusing